src/popup/router/components/AddressInput.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="address-input">
+    <InputField
+      v-model="address"
+      :placeholder="$t('pages.addressInput.placeholder')"
+      v-bind="$attrs"
+      v-on="$listeners"
+    >
+      <slot
+        v-for="slot in Object.keys($slots)"
+        :slot="slot"
+        :name="slot"
+      />
+      <Valid
+        v-if="address"
+        slot="left"
+      />
+    </InputField>
+    <button
+      v-if="!address"
+      class="right-button"
+      data-cy="scan-button"
+      @click="scan"
+    >
+      <QrScan />
+    </button>
+    <button
```

Comment:
It's not user friendly to not have paste button visible until you type somehitng. @onvisions can you elaborate on this?